oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -36,12 +38,24 @@ fn logger(pretty_name: &str, runtime: RuntimeRef, reader: ChannelReader) -> Resu
         let _ = runtime.wait_on_channels(&[Some(&reader)]);
 
         if let Some(message) = runtime.channel_read(&reader)? {
-            let message = String::from_utf8_lossy(&message.data);
-            info!("{} LOG: {}", pretty_name, message);
+            if let Ok(msg) = LogMessage::decode(&*message.data) {
+                info!(
+                    "{} LOG: {} {}:{}: {}",
+                    pretty_name,
+                    to_level_name(msg.level),
```

Comment:
Is `msg.level` not `Display` or `Debug`?

Reply:
This may be where `Prost` loses some points to `rust_protobuf`. The `enum` that `Prost` has generated for `Level` is `Debug`, but `LogMessage::level` is `i32`. So I guess this conversion is needed, unless the `#[prost(enumeration="Level", tag="3")]` annotation can do some magic that I am not aware of.